### DISCUSSION TITLE
Scheduled paused periods to enable unpaid time off

### DISCUSSION THREAD

**Discussion by PaulRBerg** [sablier-labs/flow#22] [Ideas]
## Context

We've received this feature request from Micah Zoltu today. I will share screenshots of the Discord conversation with him so I don't have to paraphrase the discussion.

<img width="400" alt="micah-1" src="https://github.com/sablier-labs/v2-open-ended/assets/8782666/5525191f-ebe4-4e57-82ec-c4fd17f2bae1">
<img width="400" alt="micah-2" src="https://github.com/sablier-labs/v2-open-ended/assets/8782666/820af3bb-1c2f-41be-be11-62063c6829bc">
<img width="400" alt="micah-3" src="https://github.com/sablier-labs/v2-open-ended/assets/8782666/c217c480-b252-40d9-9121-89030b98f884">
<img width="400" alt="micah-4" src="https://github.com/sablier-labs/v2-open-ended/assets/8782666/9e26f5ad-0bd2-4a67-b3a3-dfc70a57f3b4">
<img width="400" alt="micah-5" src="https://github.com/sablier-labs/v2-open-ended/assets/8782666/861f41cc-55fc-4b7e-b29d-1b81261d88a8">
<img width="400" alt="micah-6" src="https://github.com/sablier-labs/v2-open-ended/assets/8782666/3786f7f7-0fcd-427e-8761-9a9d1469ddd6">

## Implementation

TBD (I will edit this discussion later).

**Comment by smol-ninja** [comment]
Related
- https://github.com/sablier-labs/v2-open-ended/discussions/46

**Comment by smol-ninja** [comment]
Related issue:
- https://github.com/sablier-labs/v2-open-ended/issues/85

Since `pause()` would only pause the stream, the above objective can be achieved by setting up jobs on Gelato and calling `pause()` whenever unpaid leaves begin.

@PaulRBerg please feel free to re-open this discussion if the above comment does not address the original discussion.

**Reply by PaulRBerg** [reply]
I'm OK with the Gelato approach on the basis that this seems like a not-very-popular feature.
